### ISSUE TITLE
time should be shown in 24h format

### ISSUE THREAD

**Issue by qfina** [20c/vaping#166]
It's currently 16:12 where I am, but Vaping shows the graphs as 04:12 which reads as the morning, not the afternoon.

Apologies if I missed a configuration and this is my mistake, but I couldn't see anything.

**Comment by grizz** [comment]
I agree that sounds like a bug, thanks for reporting it! :) We'll get it on the next update.
